org.geeqie.Geeqie.json

```diff
@@ -0,0 +1,131 @@
+{
+    "app-id": "org.geeqie.Geeqie",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "geeqie",
+    "rename-desktop-file": "geeqie.desktop",
+    "rename-appdata-file": "geeqie.appdata.xml",
```

Comment:
```suggestion
```
You're already installing the appdata file with the correct name in  `post-install`